src/gui/title_bar.h

```diff
@@ -0,0 +1,30 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2023-2024  The DOSBox Staging Team
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+#ifndef DOSBOX_TITLE_BAR_H
+#define DOSBOX_TITLE_BAR_H
+
+#include "setup.h"
+
+void TITLEBAR_AddMessages();
+void TITLEBAR_AddConfig(Section_prop &secprop);
+void TITLEBAR_ReadConfig(Section* section);
```

Comment:
`std::optional<TitlebarConfig> TITLEBAR_ParseConfig(const std::string& config)` would be better, and then you could add unit tests (once you've condensed the two settings into a single format string, that is).

Reply:
But if you simplify it and parsing becomes trivial, you might not need the tests that much.

Reply:
Unfortunately, unit tests are not easy to add. Including the `sdlmain.h` leads to a lot of undefined symbols - a lot of stubbing would be needed. But I have reworked the configuration parsing, so that it just takes a string.